h2o-core/src/main/java/water/Weaver.java

```diff
@@ -63,7 +63,7 @@ static Class classForName(String className) throws ClassNotFoundException {
   }
 
   static {
```

Comment:
As I mentioned above, many of the "core" classes were touched inadvertently because of IntelliJ's auto-trim-trailing-whitespace feature. 
Although this is not very ideal, at this point it's more work to undo these changes than it's really worth it...
I'll try to watch out for these kinds of changes in the future